docs/01-prerequisites.md

```diff
@@ -47,29 +45,29 @@ systemctl enable libvirtd --now
 
 ## Libvirt CLI
 
-In order to deploy all the virtual devices needed to run the infrastructure we can make use of the virsh command line. The virsh program is the main interface for managing virsh guest domains. The program can be used to create, pause, and shutdown domains. It can also be used to list current domains.
+In order to deploy all the virtual devices needed to run the infrastructure we can make use of libvirt standard tools like virsh. The virsh program is the main interface for managing libvirt guest domains. It can be used to create, pause, shutdown or list current domains.
 
-However, I find much easier to use [kcli](https://kcli.readthedocs.io/en/master/) to deploy my virtual infrastructure.  **Kcli** is a tool meant to interact with existing virtualization providers (libvirt, kubevirt, ovirt, openstack, gcp and aws, vsphere) and to easily deploy and customize vms from cloud images. You can also interact with those vms (list, info, ssh, start, stop, delete, console, serialconsole, add/delete disk, add/delete nic,…). Futhermore, you can deploy vms using predefined profiles, several at once using plan files or entire products for which plans were already created for you.
+However, I find much easier to use [kcli](https://kcli.readthedocs.io/en/master/) to deploy my virtual infrastructure.  In author's own word, "**Kcli** is a tool meant to interact with existing virtualization providers (libvirt, kubevirt, ovirt, openstack, gcp and aws, vsphere) and to easily deploy and customize vms from cloud images."
 
 
 ### Install kcli
 
 Follow the kcli [documentation](https://kcli.readthedocs.io/en/master/#installation) to install and configure all the binaries needed to manage the libvirt daemon of the baremetal server
 
-If using CentOS, which is our case
+We will run it as a container by installing podman and launching the install script which pulls the image and sets the proper shell aliases 
 
 ```
-yum install epel-release
-yum install pip3 python3 libvirt-python libvirt-devel gcc
-pip3 install kcli
```

Comment:
Is this method not recommended anymore?

Reply:
i think this method was never really recommended, people dont like to install things through pip3 (even withina virtualenv), i would default to podman/docker install instead